tests/e2e_tests/test_data_compatibility.py

```diff
@@ -0,0 +1,150 @@
+import uuid
+
+import pytest
+import requests
+from pathlib import Path
+
+from e2e_tests.conftest import QdrantContainerConfig
+from e2e_tests.client_utils import ClientUtils
+from e2e_tests.utils import extract_archive
+
+
+class TestStorageCompatibility:
+    """Test storage and snapshot compatibility with defined previous Qdrant versions."""
+    
+    PREV_PATCH_VERSION = "v1.14.0"
+    PREV_MINOR_VERSION = "v1.15.4"
```

Comment:
I believe the patch and minor versions are accidentally flipped here.

---

Also, we need v1.14.0 for the time being, due to a bug in v1.14.1.

See [this](https://github.com/qdrant/qdrant/pull/7321/files):

```bash
PREV_PATCH_QDRANT_VERSION='v1.15.4'
PREV_MINOR_QDRANT_VERSION='v1.14.0' # skipping v1.14.1 as it contains a known data corruption (https://github.com/qdrant/qdrant/pull/6916)
```

Could you also add the same comment there, so that we're are we should pick the latest patch version again next time we update this?